frontend/src/app/main/ui/render.cljs

```diff
@@ -107,7 +107,7 @@
      #(dom/set-page-style {:size (str (mth/ceil width) "px "
                                       (mth/ceil height) "px")}))
 
-    [:& (mf/provider embed/context) {:value true}
```

Comment:
This will break import/export penpot files

Reply:
I don't think it breaks the export, export uses its own `app.main.render` ns and passes the embedd parameter explicitly.
This change does not affects the export process.